pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -173,9 +173,11 @@ def archive_pdf_file(pdf_archive_path, binary_content, extract_as_dict):
     def get_wms_url_params():
         """
         Returns the list of additionally configured wms_url_params.
+        Args:
+            params: optional argument used for given custom parameters. Expecting a dict
 
         :return: The configured wms_url_params.
-        :rtype: list
+        :rtype: dict
```

Comment:
I think it sjhould be something like the return doc in your new get_custom_wms_params function.